.gitlab-ci.yml

```diff
@@ -58,18 +58,40 @@ deploy-images-dev:
   dependencies:
     - build-images
   artifacts:
+    paths:
+      - ./env.json
     reports:
       dotenv: build.env
+  # Return testbed back to pool if the stage fails.
+  after_script:
+    - >
+      if [ $CI_JOB_STATUS != 'success' ]; then
```

Comment:
Why only return the testbed when CI_JOB_STATUS is "success"?

Reply:
!= success it is